ADONET/Program.cs

```diff
@@ -1,9 +1,8 @@
 ﻿using Microsoft.Data.Sqlite;
-using static System.Runtime.InteropServices.JavaScript.JSType;
```

Comment:
You can automatically clean these up when you save using this option:

![image](https://github.com/jcjiang/CSharp-Access-SQLite/assets/1226749/5705ddc7-40c6-4424-aa41-0983a96324a5)

Reply:
Fixed!